### DISCUSSION TITLE
Command console

### DISCUSSION THREAD

**Discussion by Rickyrvxlovet** [mpv-player/mpv#14718] [Q&A] [answered]
Hi, my question is that can we scroll in console commands? 

> `

**Comment by Bellavene** [comment]
I am also interested...

**Comment by hooke007** [comment]
If you were talking about the log history actually, it's impossible now.

**Comment by guidocella** [comment, marked as answer]
The problem with scrolling is that we can't know how many lines wrap, so page up and down would be unreliable, and are also already bound to history navigation. I can implement scrolling one line at a time with shift+up and shift+down though.
